com.unity.mobile.notifications/Editor/UnityNotificationEditorManager.cs

```diff
@@ -365,7 +354,7 @@ internal Texture2D GetDrawableResourceAssetById(string id)
 
                 if (res.Type == NotificationIconType.LargeIcon)
                 {
-                    var textXxhdpi = TextureAssetUtils.ScaleTexture(texture, (int)(192 * scale), (int)(192 * scale));
+                    TextureAssetUtils.ScaleTexture(texture, (int)(192 * scale), (int)(192 * scale));
```

Comment:
is this a typo ? shouldn't textXxhdpi should have been used instead of textXhdpi when assigning to icons below?

Reply:
Yes.. Fixed at https://github.com/Unity-Technologies/com.unity.mobile.notifications/pull/9/commits/b84a34e1be6e05eade5585b0380c4ab7695d7a5a

Reply:
Btw, you can select multiple lines https://i.stack.imgur.com/8eieb.gif for reviewing. :)